build_cmake/FrameworkOpenGl/CMakeLists.txt

```diff
@@ -6,10 +6,18 @@ set(CMAKE_MODULE_PATH
 	${CMAKE_MODULE_PATH}
 )
 
+include(Header)
+
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-if(UNIX OR WIN32 AND NOT ANDROID AND NOT APPLE AND NOT TARGET_PLATFORM_UNIX_AARCH64)
+if(TARGET_PLATFORM_ANDROID OR TARGET_PLATFORM_IOS OR TARGET_PLATFORM_UNIX_AARCH64)
```

Comment:
since we have toggle option, I think aarch64 should be removed from this condition or at least aarch 32 should be added for consistency

Reply:
I think it's a sensible default. We could add a AArch32 target later.